biocypher_metta/adapters/abc_adapter.py

```diff
@@ -76,7 +78,7 @@ def get_edges(self):
                             _target = self.hgnc_to_ensembl_map[(row[COL_DICT['target_gene']]).strip()]
                             props = {
                                 "abc_score": row[COL_DICT['abc_score']],
-                                "biological_context": row[COL_DICT['cell_type']]
+                                "biological_context": self.tissue_to_ontology_id_map[row[COL_DICT['cell_type']]]
```

Comment:
Since I've generated the `tissue_to_ontology_id_map` from the abc dataset, it already includes entries like `-Roadmap`, so there's no need to split the string further.